tests/test_external_plugins.py

```diff
@@ -43,15 +46,21 @@
 
 
 def get_parameters(plugin_filename: str):
-    try:
-        return load_test_plugin(plugin_filename).parameters
-    except ImportError:
-        return {}
+    return load_test_plugin(plugin_filename).parameters
 
 
 TEST_PLUGIN_CACHE = {}
 TEST_PLUGIN_ORIGINAL_PARAMETER_CACHE = {}
 
+# If true, copy all test Audio Units into the appropriate install path
+# before running tests. On some versions of macOS, this is necessary or
+# else Audio Units won't load.
+TEMPORARILY_INSTALL_AUDIO_UNITS = True
+PLUGIN_FILES_TO_DELETE = set()
+MACOS_PLUGIN_INSTALL_PATH = os.path.expanduser(
+    os.path.join("~", "Library", "Audio", "Plug-Ins", "Components")
```

Comment:
[Path.home](https://docs.python.org/3/library/pathlib.html#pathlib.Path.home) is the more direct way of getting the home dir and is available for python >3.5